pkg/describer/testing/integration_test_helper.go

```diff
@@ -1,41 +1,44 @@
 //go:build integration
 
-package knative_test
+package testing
 
 import (
 	"context"
 	"testing"
 	"time"
 
 	"k8s.io/apimachinery/pkg/util/rand"
-
 	fn "knative.dev/func/pkg/functions"
-	"knative.dev/func/pkg/knative"
 	"knative.dev/func/pkg/oci"
+	fntest "knative.dev/func/pkg/testing"
```

Comment:
This is "intended" to be imported as "." such that no prefix is necessary.  Not a necessary change, but for consistency that would be best.